SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -410,17 +416,14 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	// Ideally the GlobalStateManager and StaticComponentView would be created as part of USpatialWorkerConnection::Init
 	// however, this causes a crash upon the second instance of running PIE due to a destroyed USpatialNetDriver still being reference.
 	// Why the destroyed USpatialNetDriver is referenced is unknown.
-	if (Connection->GlobalStateManager == nullptr)
-	{
-		Connection->GlobalStateManager = NewObject<UGlobalStateManager>();
-	}
-	GlobalStateManager = Connection->GlobalStateManager;
+	USpatialGameInstance* GameInstance = GetGameInstance();
+	check(GameInstance != nullptr);
 
-	if (Connection->StaticComponentView == nullptr)
-	{
-		Connection->StaticComponentView = NewObject<USpatialStaticComponentView>();
-	}
-	StaticComponentView = Connection->StaticComponentView;
+	GlobalStateManager = GameInstance->GlobalStateManager;
+	check(GlobalStateManager != nullptr);
+
+	StaticComponentView = GameInstance->StaticComponentView;
```

Comment:
Can we make it a getter instead of direct acccess?

Reply:
Yes, will do it now.

Reply:
Updated.